crates/apub/src/activities/community/announce.rs

```diff
@@ -130,7 +130,7 @@ impl AnnounceActivity {
         actor: c.actor.clone().into_inner(),
         other: serde_json::to_value(c.object)?
           .as_object()
-          .expect("is object")
+          .ok_or(LemmyErrorType::NotFound)?
```

Comment:
Also unsure about this one.

Reply:
It doesnt matter because this can never really happen. Maybe add `LemmyErrorType::Unreachable` for such cases.

Reply:
Fixed.